scripts/get-report-message.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+import { command } from 'execa';
+import { readJson } from 'fs-extra';
+import { join } from 'path';
+
+type Branch = 'main' | 'next' | 'alpha';
+type Workflow = 'merged' | 'daily';
+
+const getFooter = async (branch: Branch, workflow: Workflow, job: string) => {
+  if (job === 'chromatic-sandboxes') {
+    return `\n\nThis might not necessarily be a bug, it could be a visual diff that you have to review and approve. Please check it!`;
+  }
+
+  if (branch === 'alpha') {
```

Comment:
Hey @shilman please chime in on the branch name. Thank you

Reply:
`next-release`